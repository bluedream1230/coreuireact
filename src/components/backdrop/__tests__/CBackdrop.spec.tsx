import * as React from 'react'
import { render /* ,screen */ /* ,fireEvent */ /* ,waitFor */ } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { CBackdrop } from '../../../index'

test('loads and displays CBackdrop component', async () => {
  const { container } = render(<CBackdrop>Test</CBackdrop>)
  expect(container).toMatchSnapshot()
})

test('CBackdrop customize', async () => {
  const { container } = render(<CBackdrop className="bazinga">Test</CBackdrop>)
  expect(container).toMatchSnapshot()
  // expect(container.firstChild).toHaveClass('bazinga')
  expect(container.firstChild).toHaveClass('modal-backdrop')
})
