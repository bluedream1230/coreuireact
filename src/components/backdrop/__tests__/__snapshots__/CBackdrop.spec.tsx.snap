// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CBackdrop customize 1`] = `<div />`;

exports[`loads and displays CBackdrop component 1`] = `<div />`;
